src/openforms/forms/tests/e2e_tests/test_translation_warnings.py

```diff
@@ -72,6 +72,52 @@ def setUpTestData():
 
             await expect(warning_list).not_to_be_visible()
 
+    async def test_no_warning_with_full_translations(self):
```

Comment:
I think this test doesn't add much value, the one above is sufficient IMO. You can remove it.

Reply:
Yes, I will remove it. I thought it would be more clear when I have one with a component and one without.